### ISSUE TITLE
Any news on this moving from a incubator project?

### ISSUE THREAD

**Issue by whgibbo** [spring-cloud-incubator/spring-cloud-etcd#24]

**Comment by spencergibb** [comment]
Not at the moment.

**Comment by efenderbosch** [comment]
Is there a list of items to be completed/fixed before leaving incubator status?

**Comment by spencergibb** [comment]
No. It's fallen a bit behind spring-cloud-{netflix,consul,zookeeper} mostly because of lack of bandwidth. It's not in consideration for Brixton. I'm not sure what it would take. Maybe a review and list of issues that the community could tackle. Again, that won't happen until after Brixton goes GA. /cc @mstine @viniciusccarvalho

**Comment by dsyer** [comment]
I found out this week that etcd4j only works with etcd v2. So unless there is some movement in the that library it's not clear we could support it (etcd is v3 now).

**Comment by spencergibb** [comment]
https://github.com/coreos/etcd/issues/5067

**Comment by yorksen** [comment]
So,is it means RELEASE version won't be provided in a short time until etcd4j supports etcdv3, or we find some other third-party libraries which support etcdv3 to replace etcd4j?

**Comment by dsyer** [comment]
The only way this project has a future is if there is a client to connect to the server. I don't think it's hugely difficult to write a client, so maybe someone will do that. I suppose we could cut a release with what we have, but there wouldn't be any commitment to maintaining it.

**Comment by marcingrzejszczak** [comment]
Maybe instead of releasing such an "artificial" artifact we could create a proper branch and suggest using http://jitpack.io/ or sth for people to use a "artificial milestone 1". It wouldn't be published but properly configured and could be used as a maven artifact.

**Comment by spencergibb** [comment]
I'm thinking a community maintainer that does releases to bintray/maven central, codes, PR's, issues, etc... and is managed and maintained by them.

**Comment by spencergibb** [comment]
Depends on #29
